.github/workflows/formatting.yml

```
@@ -3,15 +3,20 @@ on:
   push:
     branches:
       - main
+permissions:
+  contents: read
```

```suggestion
  contents: write
```
We cannot make this workflow read only

---

Good spot